test/source/test.ts

```diff
@@ -128,6 +128,8 @@ ava.default.after.always('evaluate Catch.reportErr errors', async t => {
     ].includes(e.message))
     // below for test "user4@standardsubdomainfes.test:8001 - PWD encrypted message with FES web portal - a send fails with gateway update error"
     .filter(e => !e.message.includes('Test error'))
+    // below for test "user@no-flags-client-configuration.flowcrypt.test - should show error when no flags is present"
+    .filter(e => !e.message.includes('Flags is required for client configuration.'))
```

Comment:
The original error should throw `UnreportableError`, then we don't have to ignore it here.